.github/workflows/linting-formatting.yaml

```diff
@@ -13,11 +13,19 @@ jobs:
     - name: Check out repo
       uses: actions/checkout@v2
 
-    - name: Install black
+    - name: Install black and tox
       run: |
         set -eux
         sudo apt-get install python3-setuptools
+        pip install tox
         pip3 install --user black==20.8b1
 
     - name: Check formatting
       run: ~/.local/bin/black --check .
+
+    - name: Linting
+      run: |
```

Comment:
What do you think about keeping the linting at the top level, so we can just run one `tox -e lint` command?